examples/misc/cursor_demo_sgskip.py

```diff
@@ -3,19 +3,20 @@
 Cursor Demo
 ===========
 
-This example shows how to use matplotlib to provide a data cursor.  It
-uses matplotlib to draw the cursor and may be a slow since this
-requires redrawing the figure with every mouse move.
+This example shows how to use Matplotlib to provide a data cursor.  It uses
+Matplotlib to draw the cursor and may be a slow since this requires redrawing
+the figure with every mouse move.
 
 Faster cursoring is possible using native GUI drawing, as in
-wxcursor_demo.py.
+:doc:`/gallery/user_interfaces/wxcursor_demo_sgskip`.
 
 The mpldatacursor and mplcursors third-party packages can be used to achieve a
 similar effect.  See
```

Comment:
As far as I understand, the "similar" part is not quite true. This example focusses on drawing a cross-hair (free or snapped to data). The mentioned cursor packages add an annotation. I think the meaning of "cursor" is quite different here and there and thus should be clarified.

Reply:
Actually the wx example linked also does something different; the whole text could be redone.  Feel free to push if you have something better.